libs/api/domains/driving-license/src/lib/util/hasResidenceHistory.ts

```
@@ -3,46 +3,25 @@ import {
-  city?: string | null
-
-  municipalityCode?: string | null
-}
+import { ResidenceEntryDto } from '@island.is/clients/national-registry-v3-applications'
 
 export const mapResidence = (
-  history: NationalRegistryResidence[] | undefined,
+  history: ResidenceEntryDto[],
 ): Residence[] => {
-  const mappedHistory = history?.map((residence) => {
+  const mappedHistory = history.map((residence) => {
```

Biome (2.4.4)

[error] This callback passed to map() iterable method should always return a value.

(lint/suspicious/useIterableCallbackReturn)